lib/src/main/java/com/futurewei/alcor/common/rbac/RbacManagerConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * Copyright 2019 The Alcor Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *         you may not use this file except in compliance with the License.
+ *         You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *         Unless required by applicable law or agreed to in writing, software
+ *         distributed under the License is distributed on an "AS IS" BASIS,
+ *         WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *         See the License for the specific language governing permissions and
+ *         limitations under the License.
+ * /
+ */
+
+package com.futurewei.alcor.common.rbac;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+
+@Configuration
+public class RbacManagerConfiguration {
+
+    @Value("${rbac.policy.type: strict}")
+    private String rbacType;
```

Comment:
What other rbacType do we plan to support?

Reply:
There are two rbacType Ignore or strict. Ignore rbac let all requestes access.

Reply:
Okay thanks. In this case, we would suggest to name the rbacType as Ignore and Enforced.